src/app/features/management/group-management-view/edit-group-view/edit-group-view.component.ts

```diff
@@ -214,6 +214,20 @@ export class EditGroupView implements OnInit, OnDestroy {
     this.router.navigate(['..'], { relativeTo: this.route }).then();
   }
 
+  /**
+   * Deletes the group.
+   */
+  delete() {
+    this.groupService.deleteGroupById(this.groupId).subscribe({
```

Comment:
das noch in das subscription array pushen => wird dann beim verlassen der Seite unsubscribed.